pilot/pkg/features/pilot.go

```diff
@@ -298,6 +298,20 @@ var (
 			"Istio Resources",
 	).Get()
 
+	AnalysisInterval = func() time.Duration {
+		val, _ := env.RegisterDurationVar(
+			"PILOT_ANALYSIS_INTERVAL",
+			10*time.Second,
+			"If analysis is enabled, pilot will run istio analyzers using this value as interval in seconds "+
+				"Istio Resources",
+		).Lookup()
+		if val < 1*time.Second {
+			log.Warnf("PILOT_ANALYSIS_INTERVAL %s is too small, it will be set to default 10 seconds", val.String())
+			return 10 * time.Second
```

Comment:
why not set to 1s?

Reply:
10s was the default. We could do 1s as well, more of a judgement call. If you would like to use 1s, we could do that.